docs/rules/no-innter-html.md

```diff
@@ -1,6 +1,6 @@
 # No `innerHTML`
 
-Using `innerHTML` poses a potential security risk. It should only be used when clearing content.
+Using `innerHTML` poses a potential security risk. Prefer using `textContent` so set text to an element.
```

Comment:
Did you mean here "_Prefer using `textContent` **to** set text to an element_"?

Reply:
```suggestion
Using `innerHTML` poses a potential security risk. Prefer using `textContent` to set text to an element.
```